### PR TITLE
Header components

### PR DESCRIPTION
**Pull request by blackinno**
- revers header component

### CODE REVIEW

File: packages/Header/Header.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="header" pt3 pb3 mb3>
```
Comment (antho1404) [inline packages/Header/Header.vue]: Same comment about the attributes `ptX`, `pbX`... than https://github.com/mesg-foundation/mesg-components/pull/58/files#r336343955

File: packages/Header/Header.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Header',
+  props: {
+    title: { type: String },
+    description: { type: String },
+    noPicture: { type: Boolean, default: false },
```
Comment (antho1404) [inline packages/Header/Header.vue]: `noPicture` could be computed based on `imgUrl` presence.
Also, one is img, the other one picture. We need consistency either both img or both picture but let's not mix

**Review by antho1404 — CHANGES_REQUESTED** [review]

**Comment by blackinno** [comment]
https://github.com/mesg-foundation/mesg-components/issues/67

**Review by antho1404 — APPROVED** [review]
Good for me, @EmmanuelDrouin can you check and merge if the design is fine

**Comment by EmmanuelDrouin** [comment]
@blackinno Can you remove the display flex on the slot? Because of the display flex the button is taking the full width of the container but it shouldn't.

**Comment by EmmanuelDrouin** [comment]
Good for me, will merge this PR

**Pull request merged**